Source/CParse/cscanner.c

```diff
@@ -350,24 +350,33 @@ static int yylook(void) {
 	}
       }
       break;
-      
+
     case SWIG_TOKEN_ELLIPSIS:
       return ELLIPSIS;
 
+    case SWIG_TOKEN_LLBRACKET:
+      do {
+        tok = Scanner_token(scan);
+      } while (tok != SWIG_TOKEN_RRBRACKET);
+      break;
+
+    case SWIG_TOKEN_RRBRACKET:
+      return RRBRACKET;
+
```

Comment:
Seems to work when I just exclude everything in cscanner.

I tried again to make changes to parse.y to actually capture the various attributes (for future use), but failed to do so